scripts/docker_build.sh

```diff
@@ -22,6 +22,7 @@ DOCKER_ARGS=$@
 
 GIT_ROOT=$(git rev-parse --show-toplevel)
 GIT_REVISION=$(git describe --abbrev=10 --always --tags --dirty)
+ORT_DOCKER=${ORT_DOCKER:-ort}
```

Comment:
This should either be dropped as we have that inline in line 30 below. Or we keep this line and just use `$ORT_DOCKER` in line 30. Such an "outlining" (opposite of "inlining") of the variable should then be done in a separate commit, though.